frontend/src/components/TaskMap.module.scss

```
@@ -7,17 +7,26 @@ $ZOOM: var(--zoom, 1);
   width: 100%;
 }
 
-.info {
-  position: relative;
-  left: 48px;
-  bottom: 26px;
-  width: 16px;
+.controlsContainer {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 80px;
 }
 
-.restart {
+.controls {
+  display: flex;
```

The controls look slightly better to me centered
```suggestion
  display: flex;
  align-items: center;
```

---

This seems good. I also changed the infoIcon to be 1px larger, so I could get rid of the `marginTop: 2px` at the control buttons - now they look nicely aligned